tiled/client/metadata_update.py

```diff
@@ -0,0 +1,60 @@
+from collections import OrderedDict
+
+# adapted from https://github.com/OpenDataServices/json-merge-patch
+
+
+class DELETE_KEY:
```

Comment:
A pattern that we use elsewhere in the codebase could be applied here: https://github.com/bluesky/tiled/blob/3b07e5f5add4d5ad2c5096a3694bee63d1a7308a/tiled/utils.py#L398-L415

Reply:
Ah.. I have seen `Sentinel` before but totally forgot of its existence! Will use that instead. But perhaps it still makes sense to import `DELETE_KEY` from `tiled.client.metadata_update`? So I'll create `DELETE_KEY` there using `Sentinel`